lib/aff4_objects/security.py

```diff
@@ -151,10 +151,11 @@ class ApprovalWithApproversAndReason(Approval):
 
   class SchemaCls(Approval.SchemaCls):
     """The Schema for the ClientAccessApproval class."""
+
     LIFETIME = aff4.Attribute(
         "aff4:approval/lifetime", rdfvalue.RDFInteger,
-        "The number of microseconds an approval is valid for.",
-        default=4 * 7 * 24 * 60 * 60 * 1000000)  # 4 weeks
+        "The number of seconds an approval is valid for.",
```

Comment:
oh you're right, it needs to be defined at import time when the class is declared.  I think the best course of action would be to get rid of the LIFETIME attribute altogether and just use the config variable.  A quick grep of the code shows this attribute isn't used anywhere else.  I suspect the intent was to allow granting of customizable approval times, but we've never used it that way.  We do lose the possibility of future flexibility if all approvals are hard-coded to use the config variable though. @grrrrrrrrr what do you think?